### PR TITLE
reverse fusion counting

### PR DESCRIPTION
**Pull request by enadeau**

### CODE REVIEW

File: tilings/strategies/fusion/constructor.py [hunk 1]
```diff
@@ -545,3 +548,188 @@ def random_sample_sub_objects(
...
+            new_param = self.forward_map(param)
+            new_value = value - child_terms[self.a_map(param)]
+            assert new_param not in terms or new_value == terms[new_param]
+            terms[new_param] = new_value
+        return terms
+
+    def get_equation(
+        self, lhs_func: sympy.Function, rhs_funcs: Tuple[sympy.Function, ...]
+    ) -> sympy.Eq:
+        raise NotImplementedError
```
Comment (christianbean) [inline tilings/strategies/fusion/constructor.py:717]: Should we make this use the same method as the normal fusion constructor?

File: tilings/algorithms/fusion.py [hunk 2]
```diff
@@ -3,9 +3,9 @@
 
 # TODO: this file needs to be type checked!
```
Comment (christianbean) [inline tilings/algorithms/fusion.py:4]: Its done, hooray!

File: tilings/strategies/fusion/constructor.py (hunk 1 above)
Comment (enadeau) [reply tilings/strategies/fusion/constructor.py:717]: Will make a fix on css to change the get equation of reverse rule to use the original rule

File: tilings/algorithms/fusion.py (hunk 2 above)
Comment (enadeau) [reply tilings/algorithms/fusion.py]: Fixed

**Comment by enadeau** [comment]
I think it is now ready to go

**Review by christianbean — APPROVED** [review]

**Pull request merged**